osu.Game.Rulesets.Osu/Objects/Drawables/Connections/FollowPoint.cs

```diff
@@ -50,6 +50,7 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
+            Delay(-TransformStartTime); // So our follow points start their transforms from t=0 (the start of the map), rather from the lead in time (where the clock is right now).

```

Comment:
trust that this is not the correct solution.

this will only work correctly if `TransformStartTime` is set correctly from an external source. we can *not* rely on that. we do not know about our parents' states.